### DISCUSSION TITLE
Ping fight

### DISCUSSION THREAD

**Discussion by KirIcare** [Crownicles/Crownicles#1846] [Idées acceptées / déjà implémentées]
Créer des mentions league
Pour l'instant pas trop d'intérêt mais quand les écarts vont se creuser, si je veux fight, je vais ping mention combats et je vais faire rappliquer des personnes avec qui je peux pas me taper.

**Comment by KirIcare** [comment]
Yo
Je relance parce que maintenant je me prends 15 à 30 pings dans la journée et 90% ne me concerne pas donc j'envisage d'enlever la mention combat.
On m'a aussi parlé de faire des mentions double league, histoire de couvrir les 400 gloire d'écart et d'avoir moins de rôle à créer.

**Reply by BastLast** [reply]
Je suis pas mega fan dvaoir 50 rôles mais je vais y réfléchir

**Reply by KirIcare** [reply]
De toute façon, si je suis le seul à trouver ça embêtant bah c'est sur que ça sert à rien de le faire

**Reply by BastLast** [reply]
> De toute façon, si je suis le seul à trouver ça embêtant bah c'est sur que ça sert à rien de le faire 

En effet c'est le genre de trucs qui va bouger que si ya bcp de votes sur la suggestion

**Comment by BastLast** [comment]
ceci a été fait au final

**Reply by Procion06** [reply]
Il manque toujours un salon pour la ligue légende

**Reply by BastLast** [reply]
C'est fait exprès

**Reply by Procion06** [reply]
il y a bien un salon ligue bois pourtant, c'est pas très logique tout ça ^^

**Reply by BastLast** [reply]
Ya plus de débutants que de pro pmayers
